manifests/03-clusterrole.yaml

```diff
@@ -316,6 +316,36 @@ roleRef:
   kind: Role
   name: insights-operator
 subjects:
-- kind: ServiceAccount
-  name: operator
+  - kind: ServiceAccount
+    name: operator
+    namespace: openshift-insights
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: openshift-insights-secrets-full-access
```

Comment:
OK this will not work. I would suggest to keep the original role names/definitions we have in 4.8 and just and the required annotations (the same as other resources have) - see https://coreos.slack.com/archives/CEGKQ43CP/p1625119637397800.